controllers/shooter.py

```diff
@@ -1,3 +1,25 @@
+from components.shooter import Shooter
+import magicbot
+from numpy import interp
+
+
 class ShooterController:
+    shooter: Shooter
+
+    # If set to true, flywheel speed is set from tunable
+    # Otherwise it is calculated from the interpolation table
+    interpolation_override = magicbot.tunable(True)
+    flywheel_speed = magicbot.tunable(0.0)
+
+    dist = magicbot.tunable(2.0)
+    ranges_xp = (2.5, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0)
+    flywheel_speed_fp = (32.0, 30.0, 36.0, 39.0, 42.0, 46.0, 51.0, 56.0)
+
     def execute(self):
-        pass
+
+        if self.interpolation_override:
+            self.shooter.motor_speed = self.flywheel_speed
```

Comment:
We should add a method to do this, but that can be done in a later PR.

Reply:
noted it in #44 for you